services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Consider initializing 'genres_ids' and 'actors_ids' with an empty list by default instead of None. This will help avoid potential issues when filtering, as passing an empty list to the '__in' filter will result in no filtering being applied.